3.10/release-notes-new-features310.md

```diff
@@ -299,6 +299,28 @@ for details.
 
 This feature is only available in the Enterprise Edition.
 
+### Optimization rule `arangosearch-constrained-sort`
+
+This new optimization rule brings significant performance improvements by 
+allowing you to perform sorting and limiting inside an ArangoSearch View
+enumeration node, if using just scoring for a sort operation.
+
+### New startup options
+
+With `arangosearch.skip-recovery`, you can skip data recovery for the specified View links
+and inverted indexes on startup. 
+Values for this startup option should have the format `<collection-name>/<link-id>`,
+`<collection-name>/<index-id>`, or `<collection-name>/<index-name>`. 
+On DB servers, the `<collection-name>` part should contain a shard name.
+
+The `arangosearch.fail-queries-on-out-of-sync` startup option allows you to
+handle queries out of sync. The option is set to `false` by default. With this setting,
```

Comment:
The sentence seems incomplete. What is out-of-sync exactly?

Based on the description in this PR https://github.com/arangodb/arangodb/pull/17046, it's limited to insert/remove operations. It sounds like you can let write operations fail if `arangosearch` View links or inverted indexes are not up-to-date with the collection data.